bases/br_ibge_censo_agropecuario/municipio/table_config.yaml

```diff
@@ -0,0 +1,1859 @@
+      description: Valor Total dos Animais
+      temporal_coverage:
+        - '1985, 2006'
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit: brl
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: valor_total_animais_transporte
+      bigquery_type: float64
+      description: Valor Total dos Animais de Transporte
+      temporal_coverage:
+        - '1985, 2006'
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit: brl
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: valor_total_maquinas_transporte
+      bigquery_type: float64
+      description: Valor Total das Maquinas de Transporte
+      temporal_coverage:
+        - '1985, 2006'
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit: brl
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: porcentagem_despesa_fertilizante
+      bigquery_type: float64
+      description: Porcentagem da despesa com fertilizantes
+      temporal_coverage: 
+        - (10)
+      covered_by_dictionary:
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit: percent
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: porcentagem_despesa_defensivos
+      bigquery_type: float64
+      description: Porcentagem da despesa com defensivos agrícolas
+      temporal_coverage:
+        - (10)
+      covered_by_dictionary:
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit: percent
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: porcentagem_despesa_sementes
+      bigquery_type: float64
+      description: Porcentagem da despesa com sementes
+      temporal_coverage: 
+        - (10)
+      covered_by_dictionary:
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit: percent
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: porcentagem_despesa_salarios
+      bigquery_type: float64
+      description: Porcentagem da despesa com salários
+      temporal_coverage: 
+        - (10)
+      covered_by_dictionary:
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit: percent
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+
```

Comment:
Verificar no documento todo 

- [ ] Alterar 1000 * brl para brl (sem esquecer de mudar o valor na coluna tbm)

- [ ] Remover aspas das descrições de colunas

- [ ] Colocar unidade ton quando for tonelada

- [ ] Tirar espaços duplos em todos 'Área  destinada a'

- [ ] Mudar porcentagem -> proporção